webpack/assets/javascripts/react_app/components/HostsIndex/Columns/core.js

```diff
@@ -173,13 +187,22 @@ const reportedDataColumns = [
     isSorted: false,
     weight: 1300,
   },
-  // { // TODO: make virtual work
-  //   columnName: 'virtual',
-  //   title: __('Virtual'),
-  //   wrapper: hostDetails => hostDetails?.reported_data?.virtual,
-  //   isSorted: false,
-  //   weight: 1400,
-  // },
+  {
+    columnName: 'virtual',
+    title: __('Virtual'),
+    wrapper: hostDetails => {
+      const value = hostDetails?.reported_data?.virtual;
+      if (value === undefined || value === null) {
+        return '';
+      }
+      if (value) {
+        return __('Virtual');
+      }
+      return __('Physical');
```

Comment:
FYI: a ternary was fine here, it's just that the linter didn't like it spread over multiple lines. So I think this would have been preferred because it's easier to follow. Now that it's merged I don't think it's important enough to go back to it though.
```suggestion
     return value ? __('Virtual') : __('Physical');
```